### PR TITLE
added TrainingSet stub

### LINKED ISSUE #5: Add TrainingSet

### PR DESCRIPTION
**Pull request by MartijnVerkleij**
fixes #5

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -83,5 +85,13 @@ def printQuantitativeBowTie(quantitativeBowTie):
     return 1
 
 
+def generate_trainingset_fault_tree(nr_events):
```
Comment (Vescatur) [inline main.py]: Inconsistent naming scheme.

File: main.py [hunk 2]
```diff
@@ -83,5 +85,13 @@ def printQuantitativeBowTie(quantitativeBowTie):
...
+    return [(x, y, random.randint(0, 1)) for x in range(0, nr_events) for y in range(0, nr_events)]
+
+
+def generate_trainingset_event_tree(nr_events):
+    return [(x, y, random.randint(0, 1)) for x in range(0, nr_events) for y in range(0, nr_events)]
```
Comment (Vescatur) [inline main.py]: Why does this methode not create a Training set object

File: trainingset.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        if training_set:
+            observations = training_set
+        elif event_tree:
+            pass
+        elif event_tree:
+            pass
+        else:
+            raise AttributeError("Fill in at least one of the arguments")
+
+    def get_observations_by_event_name(self, event_name):
```
Comment (Vescatur) [inline trainingset.py:21]: Inconsistent naming scheme

**Review by wouwouwou — DISMISSED** [review]

File: main.py (hunk 1 above)
Comment (wouwouwou) [reply main.py]: Resolved everywhere else, naming scheme is now consistent. :D

File: main.py (hunk 2 above)
Comment (wouwouwou) [reply main.py]: Resolved everywhere else, naming scheme is now consistent. :D

File: trainingset.py (hunk 3 above)
Comment (wouwouwou) [reply trainingset.py:21]: Resolved everywhere else, naming scheme is now consistent. :D

**Comment by wouwouwou** [comment]
Parameters still needs to be consistent (underscore instead of camelcase)

**Comment by wouwouwou** [comment]
Now everything is consistent 👍🏻

**Review by wouwouwou — APPROVED** [review]

**Pull request merged**